internal/server/db/backups.go

```diff
@@ -541,3 +551,204 @@ func (c *ClusterTx) RenameVolumeBackup(ctx context.Context, oldName, newName str
 
 	return nil
 }
+
+// GetStoragePoolBucketBackupsName returns the names of all backups of the storage bucket with the given name.
+func (c *Cluster) GetStoragePoolBucketBackupsName(projectName string, bucketName string) ([]string, error) {
```

Comment:
We've just gone through a recent exercise to move all our functions to `ClusterTx` instead, so this one should follow that pattern).